roles/common/tasks/networking.yml

```diff
@@ -13,6 +13,35 @@
   with_items: network_interfaces_d
   tags: ['prereboot']
 
+- name: set slrouter address
+  set_fact: sl_router_address="{{ ansible_default_ipv4.network.split('.')[0] }}.{{ ansible_default_ipv4.network.split('.')[1] }}.{{ansible_default_ipv4.network.split('.')[2] }}.{{ (ansible_default_ipv4.network.split('.')[3] | int) + 1 }}"
+  when: ansible_default_ipv4.interface == hostvars[inventory_hostname][primary_interface].device   
+
+- name: add static route for multi subnet
+  add_sys_route: state=present
```

Comment:
Why not use `command: /sbin/ip route replace {{ item }} via {{ sl_router_address }}`?